config/orm.js

```diff
@@ -0,0 +1,101 @@
+// Import MySQL connection.
+var connection = require("../config/connection.js");
+
+var orm = {
+  create: function(table, set, callback) {
+    var query = "INSERT INTO ?? SET ?";
+    var call = connection.query(query, [table, set], function(err, res) {
+      if (err) {
+        throw err;
+      }
+      callback(res)
+    });
+    console.log(call.sql);
+  },
+  read: function(from, callback) {
+    var select, where, groupBy, having, orderBy, limit, command;
+    var params = [];
+    if (typeof arguments[1] == "object") {
+      select = arguments[1].select;
+      where = arguments[1].where;
+      groupBy = arguments[1].groupBy;
+      having = arguments[1].having;
+      orderBy = arguments[1].orderBy;
+      limit = arguments[1].limit;
+      command = arguments[1].query;
+      callback = arguments[2];
+    } else {
+      callback = arguments[1];
+    }
+    if (command === true) {
+      var query = "SELECT " + from;
+    } else {
+      if (select === undefined) select = "*";
```

Comment:
This isn't bad practice but you could cover more falsy values by just making it:
```javascript
if(select)
```
that way if it is undefined, false, or null it will catch it, unless you are only concerned with it being undefined.